projects/packages/backup/src/js/hooks/useConnection.js

```diff
@@ -1,19 +1,59 @@
-import { JetpackVaultPressBackupLogo } from '@automattic/jetpack-components';
+import { JetpackVaultPressBackupLogo, Testimonials } from '@automattic/jetpack-components';
 import {
 	ConnectScreenRequiredPlan,
 	ConnectScreen,
 	CONNECTION_STORE_ID,
 } from '@automattic/jetpack-connection';
 import apiFetch from '@wordpress/api-fetch';
+import { ExternalLink } from '@wordpress/components';
 import { useSelect } from '@wordpress/data';
-import { useState, useEffect } from '@wordpress/element';
+import { useState, useEffect, createInterpolateElement } from '@wordpress/element';
 import { __ } from '@wordpress/i18n';
 import React, { useCallback } from 'react';
 import BackupPromotionBlock from '../components/backup-promotion';
 import { BackupVideoSection } from '../components/backup-video-section';
 import { WhyINeedVPBackup } from '../components/why-i-need-vp-backup';
 import { STORE_ID } from '../store';
+import benGiordanoTestimonial from './assets/ben-giordano-testimonial.png';
 import connectImage from './assets/connect-backup.png';
+import timFerrissTestimonial from './assets/tim-ferriss-testimonial.png';
+
+const testimonials = [
+	{
+		quote: __(
+			'Millions of people depend on my site, and downtime isn’t an option. Jetpack VaultPress Backup handles my site security and backups so I can focus on creation.',
+			'jetpack-backup-pkg'
+		),
+		author: 'Tim Ferriss',
+		profession: createInterpolateElement(
+			__( '<a>Author / Investor / Podcaster</a>', 'jetpack-backup-pkg' ),
+			{
+				a: <ExternalLink href="https://tim.blog/" target="_blank" rel="noreferrer noopener" />,
```

Comment:
Are we sure we want to be linking to external sources here?  And in the other slide also?

Reply:
On the Landing pages we link to these sources on jetpack.com/backup and jetpack.com/for/agencies, so I assumed we'd do that here too, but I am not sure. I can check with Simon

Reply:
Simon said to remove them: p1HpG7-kd7-p2#comment-63892 , so they are gone now and it is ready for re-review 😄